.github/workflows/nightly.yaml

```
@@ -17,3 +17,48 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           args: --namespace=test-clusters --image=cilium/cilium-test-dev:${{ steps.vars.outputs.sha_short }} "/usr/local/bin/cilium-test-gke.sh" "docker.io/cilium/cilium:latest" "docker.io/cilium/operator-generic:latest" "docker.io/cilium/hubble-relay:latest" "NightlyPolicyStress"
+  baseline-test:
+    name: Start hubble-perf test
```

Maybe just say "Start performance test" here since it's not really specific to Hubble?
```suggestion
    name: Start performance test
```